src/Command/MigrationsCommand.php

```diff
@@ -198,6 +198,7 @@ public function run(array $argv, ConsoleIo $io): ?int
         $name = explode(' ', $name);
 
         array_unshift($argv, ...$name);
+        /** @var list<string> $argv */
```

Comment:
This was the easiest way to tell ignore this since we just provide an `array` but symfony wants a `list<string>|null`